XInTheLoop/package.mo

```diff
@@ -3,6 +3,22 @@ package XInTheLoop "Components for X-in-the-Loop functionality"
   extends Icons.XInTheLoop;
     annotation(
       preferredView="info",
-      version="0.1.0",
-      uses(Modelica(version="3.2.3"), Modelica_DeviceDrivers(version="1.7.1")));
+      version="0.2.0-dev",
+      uses(Modelica(version="4.0.0"), Modelica_DeviceDrivers(version="2.1.0")),
```

Comment:
> I suggest changing this to Modelica_DeviceDrivers v2.1.1, since OM loads this version anyway.

Well - That depends on which versions the user has installed by the OM library manager.

XInTheLoop requires Modelica Standard Library v4.0.0 (currently shipped with both OM 1.19 and 1.20). XInTheLoop works with both Modelica_DeviceDrivers v2.1.0 (shipped with OM v1.19) and Modelica_DeviceDrivers v2.1.1 (shipped with OM v1.20). The user can also install both Modelica_DeviceDrivers versions at the same time - see the OM library manager. OMEdit knows that these versions are compatible and can load whichever is installed.

Reply:
My mistake. I agree, OM loads the Modelica_DeviceDriver version which is installed. Then I guess it is just a question of which version you prefer users to be on. 

A suggestion could be to note that XInTheLoop supports multiple versions in the readme. 

I think this is up to you and what you prefer.

Reply:
My statement above was perhaps not quite right. Modelica_DeviceDrivers might not be _shipped_ with any of the OM versions as described above, but rather downloaded by the OM packet manager, and therefore the default packet version depends on when running the packet manager, not only the installed OM version.

Anyway, I eventually followed your suggestion. See commit ca36ff5 with you as co-author, and resolve your two suggestions here if you agree.